testing/live/NoKillPlugin/manifest.toml

```diff
@@ -0,0 +1,5 @@
+[plugin]
+repository = "https://github.com/Bluefissure/NoKillPlugin.git"
+commit = "a361208fa5d732b5bcd31e34338c998fd1125a56"
+owners = ["Bluefissure"]
+project_path = "NoKillPlugin"
```

Comment:
You can remove this line, this refers to where your solution is stored.

Reply:
I don't think you can? I'm pretty sure it's where the .csproj is stored, not the solution.

Reply:
Oh? That's how it seemed to work when I added PingPlugin and TTT. TTT has the solution file in `src`, so I had to adjust this field. PingPlugin didn't need the field at all.